src/partials/shared-resources.json

```diff
@@ -40,5 +40,15 @@
         "provider": "[toUpper(parameters('elasticTags').provider)]"
       }
     }
-  ]
+  ],
+  "outputs": {
+    "sharedStorageAccountKey":{
+      "value": "[listKeys(resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccountName')), providers('Microsoft.Storage', 'storageAccounts').apiVersions[0]).keys[0].value]",
+      "type": "string"
+    },
+    "sharedStorageAccountSuffix": {
+      "value": "[replace(replace(reference(parameters('storageAccountName')).primaryEndpoints.blob, concat('https://', parameters('storageAccountName'), '.blob.'), ''), '/', '')]",
+      "type": "string"
+    }
```

Comment:
Want to keep the provisioning of a storage account for azure-repository plugin separate.